src/main/java/com/atlassian/jira/cloud/jenkins/common/model/PluginSiteData.java

```diff
@@ -0,0 +1,21 @@
+package com.atlassian.jira.cloud.jenkins.common.model;
+
+import java.util.Optional;
+
+public class PluginSiteData {
```

Comment:
Can you add some Javadoc comments to this to be consistent with other classes that live in the model dir?